Standards/scs-0402-v1-status-page-openapi-spec-decision.md

```diff
@@ -131,21 +131,21 @@ This means:
 
 A value of 100 is the maximum of the severity value.
 
-A severity with the value of 100 MUST always be supplied. This is the highest severity for the system. If no severity with a value of 100 exists, e.g. the highest severity value is set at 90, an `Impact` with a higher `SeverityValue` WILL be considered to be an _unkown_ severity.
+A severity with the value of 100 MUST always be supplied. This is the highest severity for the system. If no severity with a value of 100 exists, e.g. the highest severity value is set at 90, an `Impact` with a higher `SeverityValue` WILL be considered to be an _unknown_ severity.
 
 ### Component impacts
 
 Components list their impacts, which they are affected by, as read only. Only an incident creates an impact on a component. Components MUST only list their currently active impacts.
 
 ### Return of `POST` requests
 
-Generally `POST` requests create new resources. These endpoints do not return the new resource, but a unique identifier to the resource e.g. an UUID.
+Generally `POST` requests create new resources. These endpoints do not return the new resource, but a unique identifier to the resource e.g. a UUID.
 
 In most cases the new resource won't be used directly after creation. Most often list calls are used. If the new resource is used directly, it can be retrieved by the returned identifier.
 
 Payloads to POST requests SHALL NOT include ID or `Incremental` typed fields, it lies in the responsibility of the API server to assign IDs and `Incremental`s to objects.
 
-### Return of `PATCH` requestes
+### Return of `PATCH` requisites
```

Comment:
```suggestion
### Return of `PATCH` requests
```

Reply:
Updated in https://github.com/SovereignCloudStack/standards/pull/644/commits/50ab7b2c3b9d9274b29b536e69cda56c6c5c0a9b